src/app/switcher/switcher.service.ts

```diff
@@ -15,9 +15,10 @@ const api = {
 };
 
 export interface ProgramObj {
-  program: Program;
-  project: Project;
-  timeline: Timeline;
+  program: Program,
```

Comment:
@zawmyohtut for interface, the comma should be semicolon ";"  it's not object.

Reply:
@trtshen hi chaw, I have updated. and will merge the PR now.